addons/sys_gestures/CfgGesturesMale.hpp

```diff
@@ -0,0 +1,72 @@
+class CfgGesturesMale {
+    skeletonName = "OFP2_ManSkeleton";
+    class Default;
+    class States {
+        class acre_radio_base: Default {
+            canPullTrigger = 1;
+            canReload = 0;
+            connectAs = "";
+            connectFrom[] = {};
+            connectTo[] = {};
+            disableWeapons = 0;
+            disableWeaponsLong = 0;
+            enableBinocular = 1;
+            enableMissile = 1;
+            enableOptics = 1; //was 0
+            equivalentTo = "";
+            forceAim = 0;
+            headBobMode = 0;
+            headBobStrength = 0;
+            interpolateFrom[] = {};
+            interpolateTo[] = {};
+            interpolateWith[] = {};
+            interpolationRestart = 0;
+            interpolationSpeed = 6;
+            limitGunMovement = 0;
+            looped = 0;
+            minPlayTime = 0.5;
+            preload = 1;
+            ragdoll = 0;
+            relSpeedMax = 1;
+            relSpeedMin = 1;
+            showHandGun = 0;
+            showItemInHand = 0;
+            showItemInRightHand = 0;
+            showWeaponAim = 1;
+            soundEdge[] = {0.5,1};
+            soundEnabled = 1;
+            soundOverride = "";
+            speed = 0.3;
+            static = 0;
+            terminal = 0;
+            Walkcycles = 1;
+            weaponLowered = 0;
+            leftHandIKBeg = 0;
+            leftHandIKCurve[] = {0};
+            leftHandIKEnd = 0;
+            rightHandIKBeg = 1;
+            rightHandIKCurve[] = {1};
+            rightHandIKEnd = 1;
+        };
+        class acre_radio_helmet: acre_radio_base {
+            file = "a3\anims_f_epa\data\anim\sdr\cts\custom\a_in\acts_listeningtoradioloop.rtm";
+            minPlayTime = 2;
+            mask = "acre_UpperBodyNoRArm";
+        };
+        class acre_radio_vest: acre_radio_base {
+            file = "a3\anims_f_bootcamp\data\anim\sdr\cts\acts_kore_talkingoverradio_loop.rtm";
+            mask = "acre_UpperBodyNoRArm";
+        };
+        class GestureNod;
+        class acre_radio_stop: GestureNod {
+            file = "a3\anims_f\data\anim\sdr\gst\gestureEmpty.rtm";
+            disableWeaponsLong = 0;
+            enableBinocular = 1;
+            enableMissile = 1;
+            enableOptics = 1;
+        };
```

Comment:
This gesture fill cause annoying camera shake/nodding on radio stop.
```suggestion
        class acre_radio_stop: GestureNod {
            file = "a3\anims_f\data\anim\sdr\gst\gestureEmpty.rtm";
            disableWeapons = 0;
            disableWeaponsLong = 0;
            enableBinocular = 1;
            enableMissile = 1;
            enableOptics = 1;
            mask = "empty";
        };
```